src/main/drivers/timer.c

```diff
@@ -655,6 +655,12 @@ _TIM_IRQ_HANDLER(TIM4_IRQHandler, 4);
 #if USED_TIMERS & TIM_N(5)
 _TIM_IRQ_HANDLER(TIM5_IRQHandler, 5);
 #endif
+#if USED_TIMERS & TIM_N(6)
+_TIM_IRQ_HANDLER(TIM6_IRQHandler, 6);
```

Comment:
IRQ handlers on ARM are standard functions. There is nothing magic when defining them, TIM6_IRQHandler is standard, but unused, function.

`weak` only provides default handler when user supplied one is not available